modules/core/src/test/java/org/apache/ignite/internal/GridMBeansTest.java

```diff
@@ -41,9 +41,39 @@
     /** Executor name for setExecutorConfiguration */
     private static final String CUSTOM_EXECUTOR_1 = "Custom executor 1";
 
+    /** For storing original value of IGNITE_MBEAN_APPEND_CLASS_LOADER_ID. */
+    private String cldProp;
+
+    /** For storing original value of IGNITE_MBEAN_APPEND_JVM_ID. */
+    private String jvmIdProp;
+
     /** Create test and auto-start the grid */
     public GridMBeansTest() {
-        super(true);
+        super(false);
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Minor: can be a one-line doc (`/** {@inheritDoc} */`)

Reply:
+